customer-terminate-service/domain/src/test/java/se/inera/intyg/cts/domain/model/TerminationBuilderTest.java

```diff
@@ -88,12 +103,24 @@ void shallNotExceptExistingTerminationWithoutCreated() {
       assertEquals("Missing Created", exception.getMessage());
     }
 
+    @Test
+    void shallNotExceptExistingTerminationWithoutModified() {
```

Comment:
shallNotAccept... ? Also the above and below method :)

Reply:
Fixed!